Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -742,8 +744,7 @@ class BrowserViewController: UIViewController, SearchBarLocationProvider, Themea
 
         if CoordinatorFlagManager.isCoordinatorEnabled {
             NSLayoutConstraint.activate([
-                contentContainer.topAnchor.constraint(equalTo: statusBarOverlay.bottomAnchor, priority: .init(998)),
-                contentContainer.topAnchor.constraint(equalTo: header.bottomAnchor, priority: .init(999)),
```

Comment:
This was done to fix the previous method of handling the status bar overlay (see next comment)